Add AUTO_START parameter support for non-CLI (#170)
NoiseByNorthwest/php-spx: README.md

SPX_ENABLED=1 SPX_REPORT=full SPX_AUTO_START=0 my_script.php
```

Automatic start can also be disabled for web requests via the `spx.http_profiling_auto_start` INI parameter or via the control panel.

Side notes:
- `spx_profiler_start()` and `spx_profiler_stop()` can safely be nested.

> - When profiling with the _full_ report type, `spx_profiler_stop()` returns the report key so that you will be able to store it somewhere, for instance among other information related to the profiled span. With the report key you can build the analysis screen URL which ends with this pattern `/?SPX_UI_URI=/report.html&key=<report key>`.  

Author: ```suggestion
- when profiling with the _full_ report type, `spx_profiler_stop()` returns the report key so that you will be able to store it somewhere, for instance among other information related to the profiled span. With the report key you can build the analysis screen URL which ends with this pattern `/?SPX_UI_URI=/report.html&key=<report key>`.  
```

> - in non-CLI (web request) context or in CLI context with the _full_ report type, `spx_profiler_stop()` returns the report key so that you will be able to store it somewhere, for instance among other information related to the profiled span. With the report key you can build the analysis screen URL which ends with this pattern `/?SPX_UI_URI=/report.html&key=<report key>`.  

Reviewer: As non-CLI context and CLI context are the only contexts, you could dispense with the modifiers and just say `when running with the _full_ report type`

Author: ```suggestion
- When profiling with the _full_ report type, `spx_profiler_stop()` returns the report key so that you will be able to store it somewhere, for instance among other information related to the profiled span. With the report key you can build the analysis screen URL which ends with this pattern `/?SPX_UI_URI=/report.html&key=<report key>`.  
```
